packages/create-vite/template-svelte-ts/src/App.svelte

```diff
@@ -1,65 +1,42 @@
 <script lang="ts">
-  import logo from './assets/svelte.png'
+  import viteLogo from '/vite.svg'
+  import svelteLogo from './assets/svelte.svg'
   import Counter from './lib/Counter.svelte'
 </script>
 
 <main>
-  <img src={logo} alt="Svelte Logo" />
-  <h1>Hello Typescript!</h1>
-
-  <Counter />
+  <div>
+    <a href="https://vitejs.dev"> 
+      <img src={viteLogo} alt="Vite Logo" />
+    </a>
+    <a href="https://svelte.dev"> 
+      <img src={svelteLogo} alt="Svelte Logo" />
+    </a>
```

Comment:
```suggestion
    <a href="https://vitejs.dev" target="_blank"> 
      <img src={viteLogo} alt="Vite Logo" />
    </a>
    <a href="https://svelte.dev" target="_blank"> 
      <img src={svelteLogo} alt="Svelte Logo" />
    </a>
```
Other templates have `target="_blank"` on links, so it might be better to have it here too.